common/flatpak-repo-utils-private.h

```diff
@@ -80,6 +80,11 @@
 
 #define FLATPAK_SUMMARY_HISTORY_LENGTH_DEFAULT 16
 
+#define FLATPAK_SPARSE_CACHE_KEY_ENDOFLINE "eol"
+#define FLATPAK_SPARSE_CACHE_KEY_ENDOFLINE_REBASE "eolr"
+#define FLATPAK_SPARSE_CACHE_KEY_TOKEN_TYPE "tokt"
+#define FLATPAK_SPARSE_CACHE_KEY_EXTRA_DATA_SIZE "eds"
```

Comment:
Those exact defines already exist just above.

Reply:
That commit was left over from a previous rebase and now redundant, I dropped it.